plugins/tech-insights-backend/package.json

```diff
@@ -41,15 +41,17 @@
     "@backstage/errors": "^0.2.2",
     "@backstage/plugin-tech-insights-common": "^0.2.3",
     "@backstage/plugin-tech-insights-node": "^0.2.6",
+    "@backstage/backend-tasks": "^0.1.10",
     "@types/express": "^4.17.6",
+    "cron": "^1.8.2",
```

Comment:
Simply because of the CronTime type which allowed me to get the next execution time of the cron schedule without actually scheduling the task. I didn't see a way in node-cron to get the same type of value.